### PR TITLE
#1102 When pasting Extended SMILES structure with stereochemistry there are two &1 centers instead of an ABS and an &1

### PR DESCRIPTION
**Pull request by Wadym**

### CODE REVIEW

File: core/indigo-core/molecule/src/smiles_loader.cpp [hunk 1]
```diff
@@ -3129,7 +3134,7 @@ void SmilesLoader::_readAtom(Array<char>& atom_str, bool first_in_brackets, _Ato
                 subatom = std::make_unique<QueryMolecule::Atom>(QueryMolecule::ATOM_NUMBER, element);
             else
             {
-                if (element_assigned)
+                if (element_assigned && !stereochemistry_options.ignore_errors)
```
Comment (AlexanderSavelyev) [inline core/indigo-core/molecule/src/smiles_loader.cpp:3137]: please add python test for this change

File: bingo/oracle/src/oracle/ora_wrap.cpp [hunk 2]
```diff
@@ -599,7 +599,10 @@ bool OracleStatement::executeSingleInt(int& result, OracleEnv& env, const char*
     statement.prepare();
     statement.defineIntByPos(1, &result);
     if (!statement.executeAllowNoData())
+    {
```
Comment (AlexanderSavelyev) [inline bingo/oracle/src/oracle/ora_wrap.cpp:602]: move this change to separate PR

**Review by AlexanderSavelyev — CHANGES_REQUESTED** [review]

File: core/indigo-core/molecule/smiles_loader.h [hunk 3]
```diff
@@ -187,6 +190,7 @@ namespace indigo
 
     private:
         SmilesLoader(const SmilesLoader&); // no implicit copy
+        std::unordered_set<int> _overtly_defined_abs;
```
[the remark this answers is not in the document]
Comment (Wadym) [reply core/indigo-core/molecule/smiles_loader.h]: As we have discussed we are not going to track changes attributes during SMILES parsing, hence the container is going to be moved into the method.

File: core/indigo-core/molecule/src/smiles_loader.cpp (hunk 1 above)
Comment (Wadym) [reply core/indigo-core/molecule/src/smiles_loader.cpp:3140]: As per company policy, bugs unrelated to the issue should not be included in the PR. The commit is going to be reverted

**Review by AlexanderSavelyev — APPROVED** [review]
